`NoClassDefFoundError` during data batch processing

## Description
<!-- Describe your issue -->
An error occurs during data batch processing in the `WordLearningEventImportScheduler`:

```
[${web:attr.content_language}] 11:30:00.001 [pool-2-thread-1] INFO  ai.elimu.tasks.analytics.WordLearningEventImportScheduler - csvFile: /root/.elimu-ai/lang-THA/analytics/android-id-5b7c682a12ecbe2e/version-code-3001030/word-learning-events/5b7c682a12ecbe2e_3001030_word-learning-events_2025-03-18.csv
[${web:attr.content_language}] 11:30:00.004 [pool-2-thread-1] INFO  ai.elimu.util.csv.CsvAnalyticsExtractionHelper - extractWordLearningEvents
[${web:attr.content_language}] 11:30:00.007 [pool-2-thread-1] INFO  ai.elimu.util.csv.CsvAnalyticsExtractionHelper - csvFilePath: /root/.elimu-ai/lang-THA/analytics/android-id-5b7c682a12ecbe2e/version-code-3001030/word-learning-events/5b7c682a12ecbe2e_3001030_word-learning-events_2025-03-18.csv
[${web:attr.content_language}] 11:30:00.007 [pool-2-thread-1] INFO  ai.elimu.util.csv.CsvAnalyticsExtractionHelper - header: [id, timestamp, android_id, package_name, word_id, word_text, learning_event_type, additional_data]
[${web:attr.content_language}] 11:30:00.008 [pool-2-thread-1] ERROR org.springframework.scheduling.support.TaskUtils$LoggingErrorHandler - Unexpected error occurred in scheduled task
java.lang.NoClassDefFoundError: org/apache/commons/io/input/UnsynchronizedBufferedReader
        at org.apache.commons.csv.CSVParser.<init>(CSVParser.java:561) ~[commons-csv-1.14.0.jar:1.14.0]
        at org.apache.commons.csv.CSVParser.<init>(CSVParser.java:526) ~[commons-csv-1.14.0.jar:1.14.0]
        at org.apache.commons.csv.CSVParser.<init>(CSVParser.java:497) ~[commons-csv-1.14.0.jar:1.14.0]
        at ai.elimu.util.csv.CsvAnalyticsExtractionHelper.extractWordLearningEvents(CsvAnalyticsExtractionHelper.java:53) ~[classes/:?]
        at ai.elimu.tasks.analytics.WordLearningEventImportScheduler.execute(WordLearningEventImportScheduler.java:75) ~[classes/:?]
        at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke0(Native Method) ~[?:?]
        at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:77) ~[?:?]
        at java.base/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43) ~[?:?]
        at java.base/java.lang.reflect.Method.invoke(Method.java:569) ~[?:?]
        at org.springframework.scheduling.support.ScheduledMethodRunnable.run(ScheduledMethodRunnable.java:84) ~[spring-context-6.0.9.jar:6.0.9]
        at org.springframework.scheduling.support.DelegatingErrorHandlingRunnable.run(DelegatingErrorHandlingRunnable.java:54) [spring-context-6.0.9.jar:6.0.9]
        at org.springframework.scheduling.concurrent.ReschedulingRunnable.run(ReschedulingRunnable.java:96) [spring-context-6.0.9.jar:6.0.9]
        at java.base/java.util.concurrent.Executors$RunnableAdapter.call(Executors.java:539) [?:?]
        at java.base/java.util.concurrent.FutureTask.run(FutureTask.java:264) [?:?]
        at java.base/java.util.concurrent.ScheduledThreadPoolExecutor$ScheduledFutureTask.run(ScheduledThreadPoolExecutor.java:304) [?:?]
        at java.base/java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1136) [?:?]
        at java.base/java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:635) [?:?]
        at java.base/java.lang.Thread.run(Thread.java:840) [?:?]
Caused by: java.lang.ClassNotFoundException: org.apache.commons.io.input.UnsynchronizedBufferedReader
```

The file contains the following headers:
```csv
id,time,android_id,package_name,word_id,word_text,learning_event_type
143,1742293958238,5b7c682a12ecbe2e,ai.elimu.vitabu.debug,36,ฉัน,WORD_PRESSED
```

## Why?
<!-- Explain _why_ this issue is needed -->

This issue is needed because it's blocking the [`ml-datasets`](https://github.com/elimu-ai/ml-datasets)
 from accessing the datasets per student.
---

> elimu.ai's mission is to build innovative learning software that empowers out-of-school children to teach themselves basic reading📖, writing✍🏽 and math🔢 **within 6 months**.

--- a/pom-dependency-tree.txt
+++ b/pom-dependency-tree.txt
@@ -1,4 +1,4 @@
-ai.elimu:webapp:war:2.6.11-SNAPSHOT
+ai.elimu:webapp:war:2.6.12-SNAPSHOT
 +- ai.elimu:model:jar:model-2.0.97:compile
 |  \- com.google.code.gson:gson:jar:2.13.0:compile
 |     \- com.google.errorprone:error_prone_annotations:jar:2.37.0:compile
@@ -20,13 +20,13 @@ ai.elimu:webapp:war:2.6.11-SNAPSHOT
 +- org.apache.logging.log4j:log4j-slf4j2-impl:jar:2.24.3:runtime
 |  \- org.slf4j:slf4j-api:jar:2.0.16:compile
 +- commons-lang:commons-lang:jar:2.6:compile
-+- commons-io:commons-io:jar:2.16.1:compile
 +- commons-validator:commons-validator:jar:1.9.0:compile
 |  +- commons-beanutils:commons-beanutils:jar:1.9.4:compile
 |  +- commons-digester:commons-digester:jar:2.1:compile
 |  +- commons-logging:commons-logging:jar:1.3.2:compile
 |  \- commons-collections:commons-collections:jar:3.2.2:compile
 +- org.apache.commons:commons-csv:jar:1.14.0:compile
+|  +- commons-io:commons-io:jar:2.18.0:compile
 |  \- commons-codec:commons-codec:jar:1.18.0:compile
 +- org.pmml4s:pmml4s_3:jar:1.5.6:compile
 |  +- org.scala-lang:scala3-library_3:jar:3.1.3:compile

--- a/pom.xml
+++ b/pom.xml
@@ -373,11 +373,6 @@
       <version>2.6</version>
     </dependency>
     <dependency>
-      <groupId>commons-io</groupId>
-      <artifactId>commons-io</artifactId>
-      <version>2.16.1</version>
-    </dependency>
-    <dependency>
       <groupId>commons-validator</groupId>
       <artifactId>commons-validator</artifactId>
       <version>1.9.0</version>

--- a/src/main/webapp/WEB-INF/jsp/analytics/students/id.jsp
+++ b/src/main/webapp/WEB-INF/jsp/analytics/students/id.jsp
@@ -1,5 +1,5 @@
 <content:title>
-    🎓 Student ID ${student.id}
+    🎓 Student ${student.id}
 </content:title>
 
 <content:section cssId="studentPage">

--- a/src/main/webapp/WEB-INF/jsp/analytics/students/list.jsp
+++ b/src/main/webapp/WEB-INF/jsp/analytics/students/list.jsp
@@ -3,6 +3,11 @@
 </content:title>
 
 <content:banner>
+    <style>
+        #index-banner {
+            background-position: 0 75%;
+        }
+    </style>
     <br />
     <br />
     <br />
@@ -11,14 +16,9 @@
     <br />
     <br />
     <br />
-    <br />
-    <br />
-    <br />
-    <br />
-    <br />
-    <br />
-    <br />
-    <br />
+    <div class="row center">
+        <h1 class="header center white-text">Students<span style="color: #A59ADD;">(${fn:length(students)})</span></h1>
+    </div>
 </content:banner>
 
 <content:section cssId="studentsListPage">
@@ -45,3 +45,13 @@
         </table>
     </div>
 </content:section>
+
+<content:aside>
+    <div class="card-panel deep-purple lighten-5">
+        <h5 class="center">Sponsors 🫶🏽</h5>
+        <p>
+            To become a sponsor of one of these students, join our  
+            <a href="https://sponsors.elimu.ai" target="_blank">sponsorship program</a>.
+        </p>
+    </div>
+</content:aside>

